redwood-layout-uiview/src/commonMain/kotlin/app/cash/redwood/layout/uiview/YogaUIView.kt

```diff
@@ -96,6 +95,24 @@ internal class YogaUIView(
   }
 
   private fun calculateLayoutWithSize(size: CValue<CGSize>): CValue<CGSize> {
+    return size.useContents {
+      calculateLayoutWithSize(width.toYoga(), height.toYoga())
+    }
+  }
+
+  private fun calculateLayoutWithSize(width: Float, height: Float): CValue<CGSize> {
+    rootNode.requestedWidth = when (widthConstraint) {
+      Constraint.Fill -> width
+      else -> Size.UNDEFINED
+    }
+    rootNode.requestedMaxWidth = width
+
+    rootNode.requestedHeight = when (heightConstraint) {
+      Constraint.Fill -> height
+      else -> Size.UNDEFINED
+    }
+    rootNode.requestedMaxHeight = height
```

Comment:
Previously we were never setting these on iOS! But we were setting them on iOS. Now we’re consistent.